scripts/exregional_run_post.sh

```diff
@@ -365,8 +365,16 @@ fi
 #   clean forecast netcdf files for saving space
 #-----------------------------------------------------------------------
 #
-rm -f ${dyn_file}
-rm -f ${phy_file}
+indx="00 06 12 18"
+for i in $indx
+do
+  if [ "$cyc" == $i ]; then
+    echo "long forecast cycle, keep .nc for bufrsnd" 
+  else
+    rm -f ${dyn_file}
+    rm -f ${phy_file}
+  fi
+done
```

Comment:
Sometimes run_post jobs might not be recognized when it actually failed. In this case we might need to rerun post jobs and therefore we still need the dynf*nc and phyf*nc files.

Reply:
This is for real-time NA 3km. I added a check to only run this part for NA 3km grid.